src/core/jitter.cpp

```diff
@@ -106,9 +130,17 @@ std::unique_ptr<MemoryBuffer> Jitter::jitAsm(const std::string &Asm,
   IRB.CreateCall(FType, RawAsm);
   IRB.CreateRetVoid();
 
+  llvm::Triple TT = llvm::Triple(Triple);
+  if (TT.getArch() == llvm::Triple::ArchType::arm) {
+      initializeARMAssembler();
+  } else if (TT.getArch() == llvm::Triple::ArchType::aarch64) {
+      initializeAArch64Assembler();
```

Comment:
How about the following, and calling it in the constructor (wouldn't bother lazily initializing this in jitAsm)?
```cpp
void Jitter::initializeArchTarget() {
  if (!HasTargetInitialized) {
    llvm::Triple TT(Triple);
    if (TT.isARM()) initializeARMAssembler();
    else if (TT.isAArch64()) initializeAArch64Assembler();
    else if (TT.isX86()) initializeX86Assembler();
    else fatalError(fmt::format("Unsupported arch type: {}", TT.getArch()));
    HasTargetInitialized = true;
  }
}
```